README.rst

```diff
@@ -60,11 +60,13 @@ Development
 
     pip install -r requirements/default.txt
 
-**Step 3.** Install fury via::
+**Step 3.** Install fury 
+
+As `Local Project Install <https://pip.pypa.io/en/stable/reference/pip_install/#id44>`__ using::
```

Comment:
Can you use lowercase?  `local project install`

Reply:
sure thing !